src/Themes/NexusMods.Themes.NexusFluentDark/Extensions/ThicknessExtension.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Avalonia;
+using Avalonia.Markup.Xaml;
+using Avalonia.Media;
+using JetBrains.Annotations;
+
+namespace NexusMods.Themes.NexusFluentDark.Extensions;
+
+/// <summary>
+/// Xaml extension for Thickness (used by Margin, Padding, BorderThickness).
+/// This allows setting a Thickness using a resource for each side.
+/// </summary>
+/// <example>
+///     <Border Padding="{extensions:Thickness Left={StaticResource Spacing-2}, Top=0, Right={StaticResource Spacing-2}, Bottom=0}"/>
+/// </example>
+public class ThicknessExtension: MarkupExtension
```

Comment:
Uhh exciting. It is going to be a bit wordy to use fur sure, but still better than hardcoded values. 
Updating all occurrences to this might be another fun thing to do during winter holidays.